src/main.js

```diff
@@ -134,10 +137,19 @@ let load = base.loader({
       let regions = cfg.backend.s3.regions.split(',');
 
       for (let region of regions) {
-        let bucket = cfg.backend.s3.bucketBase + cfg.server.env + '-' + region;
+        let bucket;
+        if (profile === 'test') {
+          if (!testBucket) {
+            testBucket = uuid.v4().replace(/-/g, '');
+          }
```

Comment:
I do delete them by default.  We did do this with s3-copy-proxy.  The problem I have with statically named buckets is that I found each run of the unit tests would be affected by the previous run.  I can't remember if deletes in us-west-1 are immediately consistent or not.  If they are, I guess I could have a statically named bucket and do a recursive delete from the bucket after each run...  Easy to change this behaviour after this code lands and we want to change it, so I'm not going to block on this change.